common-utils/src/main/java/eu/arrowhead/common/mqtt/filter/authorization/BlacklistMqttFilter.java

```diff
@@ -0,0 +1,27 @@
+package eu.arrowhead.common.mqtt.filter.authorization;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.stereotype.Service;
+
+import eu.arrowhead.common.Constants;
+import eu.arrowhead.common.mqtt.filter.ArrowheadMqttFilter;
+import eu.arrowhead.common.mqtt.model.MqttRequestModel;
+
+@Service
+@ConditionalOnExpression("${mqtt.api.enabled:true} && ${enable.blacklist.filter:true}")
```

Comment:
defaults should be false

Reply:
Don't get it. This bean will be instantiated only if both prop is true.